src/locale/locales/tr/messages.po

```diff
@@ -1058,7 +1058,7 @@ msgstr ""
 
 #: src/screens/StarterPack/StarterPackScreen.tsx:610
 #~ msgid "Are you sure you want delete this starter pack?"
-#~ msgstr ""
+#~ msgstr "Bu başlangıç paketini silmek istediğinize emin misiniz?"
```

Comment:
A friendly tip from a fellow translator: there's no need to update strings where the line starts with `#~`. These are obsolete strings that are no longer used in the app but haven't been removed from the message catalog yet. Some translators choose to remove them to keep things tidy, but it's not necessary to do that if you prefer not to as they will simply be ignored.